eldritch/encounters.py

```diff
@@ -367,6 +367,136 @@ def Isle7(char):
   clues = events.Gain(char, {"clues": 2})
   return events.PassFail(char, check, clues, events.Nothing())
 
+def Hospital1(char):
+  return events.Loss(char, {"clues": 1})
```

Comment:
This should be conditioned on the character having at least one clue (otherwise, they could go negative).